ruby.hcl

```diff
@@ -3,6 +3,11 @@ strip = 2
 binaries = ["bin/*"]
 test = "ruby --version"
 
+env = {
+  GEM_HOME: "${HERMIT_ENV}/.hermit/ruby/gem",
+  PATH: "${GEM_HOME}/bin:${PATH}",
```

Comment:
@aat may I refer to the above defined `${GEM_HOME}` here? It works locally but maybe that's undefined behaviour?

Reply:
Ah I see what you were asking about. This will actually work, because Hermit iteratively resolves envars.

Reply:
Iteratively as in it recognises the dependencies between them?

Reply:
Will https://github.com/cashapp/hermit-packages/pull/185 then?